support/modules/rootfs-image

```diff
@@ -101,6 +122,7 @@ ensure_correct_root_mounted() {
 
 check_requirements() {
     parse_conf_file
+    check_environment_canary
```

Comment:
We should only warn, no hard exits?

Reply:
`set -e` is set, so this is a hard exit.